client/blocks/post-likes/index.jsx

```diff
@@ -91,7 +91,7 @@ class PostLikes extends PureComponent {
 
 		return (
 			<div className={ classes } { ...extraProps }>
-				{ !! postId && <QueryPostLikes siteId={ siteId } postId={ postId } needsLikers={ true } /> }
```

Comment:
You can remove the `needsLikers` prop and related code from the component completely now because it will always be false :)

Reply:
You're right! I'll tackle that in a follow-up PR once this one ships.

Reply:
Follow up PR to remove the `needsLikers` prop: https://github.com/Automattic/wp-calypso/pull/84538